tests/artist-update-patch.test.js

```diff
@@ -0,0 +1,41 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Update Artist', () => {
+  let artist;
+  beforeEach(async () => {
+    const { rows } = await db.query(
+      'INSERT INTO Artists (name, genre) VALUES( $1, $2) RETURNING *',
+      ['Tame Impala', 'rock']
+    );
+
+    artist = rows[0];
+  });
+
+  describe('PATCH /artists/{id}', () => {
+    it('updates the artist and returns the updated record', async () => {
+      const { status, body } = await request(app)
+        .patch(`/artists/${artist.id}`)
+        .send({ name: 'something different', genre: 'rock' });
+
+      expect(status).to.equal(200);
+
+      expect(body).to.deep.equal({
+        id: artist.id,
+        name: 'something different',
+        genre: 'rock',
+      });
+    });
+
```

Comment:
I would add additional tests here to check the patch works if only sending either the name or genre